build/commands/lib/config.js

```diff
@@ -30,7 +31,7 @@ const run = (cmd, args = [], options) => {
 }
 
 // this is a huge hack because the npm config doesn't get passed through from brave-browser .npmrc/package.json
-var packageConfig = function (key, sourceDir = rootDir) {
+var packageConfig = function (key, sourceDir = braveCoreDir) {
```

Comment:
I think this method got confused at some point, the only reason we needed it at all was for brave-browser. See the comment above. If we are only reading from brave-core we don't need this hack.

Reply:
@petemill can you confirm please?

Reply:
Things have changed significantly since the original method so let's just remove the comment here since it no longer applies in any case

Reply:
Removed comment, thank you.